src/ast/async_event_types.h

```diff
@@ -76,6 +76,20 @@ struct Time
   }
 } __attribute__((packed));
 
+struct Strftime
+{
+  uint64_t strftime_id;
+  uint64_t nsecs_since_boot;
+
+  std::vector<llvm::Type*> asLLVMType(ast::IRBuilderBPF& b)
+  {
+    return {
+      b.getInt64Ty(), // strftime id
```

Comment:
```suggestion
      b.getInt64Ty(), // asyncid
```
nit: to keep it consistent with the othrs

Reply:
Actually, it's not `asyncid` since `strftime()` doesn't call `CreatePerfEventOutput` directly and we don't need the `asyncid` to indicate what the data is in `perf_event_printer`. Instead, it's the index in the `bpftrace.strftime_args_` which stores the format strings of strftime.